doc/030_preparing_a_new_repo.rst

```diff
@@ -568,6 +568,11 @@ The number of concurrent connections to the Azure Blob Storage service can be se
 ``-o azure.connections=10`` switch. By default, at most five parallel connections are
 established.
 
+The access tier of the blobs uploaded to the Azure Blob Storage service can be set with the
+``-o azure.access-tier=Cool`` switch. The allowed values are ``Hot``, ``Cool``, ``Cold`` or
+``Archive``. If unspecified, the default is inferred from the default configured on the 
```

Comment:
archive storage is not and cannot be officially supported at the moment. This must be removed from the docs.

Reply:
Resolved. I have removed the mention of the `Archive` tier from the docs. In the changelog, I have mentioned that `Archive` tier is not officially supported; referencing the changelog entry https://github.com/restic/restic/pull/4584/files